spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/keras/KerasLayer.scala

```diff
@@ -51,23 +67,94 @@ object KerasLayerSerializer extends ContainerSerializable with TKerasSerializerH
   }
 }
 
+private[bigdl] class KerasIdentityActivationWrapper[T: ClassTag]
```

Comment:
This is somewhat complex. We just need a KerasWrapper, and this can be implemented as KSequential(KerasWrapper(layer), activation), yes?

KerasIdentityWrapper can simply extend KerasWrapper.

Reply:
There's a little bit overhead for `KerasLayerWrapper ` as it would need to `forward` once to get the `outputshape` although ideally it can wrap any Torch Layer.